core/core_bind.cpp

```diff
@@ -1374,9 +1374,9 @@ Vector<String> _File::get_csv_line(const String &p_delim) const {
 	return f->get_csv_line(p_delim);
 }
 
-/**< use this for files WRITTEN in _big_ endian machines (ie, amiga/mac)
+/**< use this for files WRITTEN in _big_ endian machines (i.e., amiga/mac)
```

Comment:
If we're nitpicking there, I would also remove the unnecessary comma.

Reply:
Unrelated note: `mac` is reference to 68k and PPC macs, which might be confusing. Better to use some more up-to-date example, like TCP/IP.

Reply:
> Unrelated note: `mac` is reference to 68k and PPC macs, which might be confusing. Better to use some more up-to-date example, like TCP/IP.

I think I can append TCP/IP as examples (or should I replace them?)
Or leave as it is....